weave-js/src/components/PagePanelComponents/Home/Browse3/pages/CallPage/OverflowMenu.tsx

```diff
@@ -6,27 +6,39 @@ import {
 } from '@material-ui/core';
 import {PopupDropdown} from '@wandb/weave/common/components/PopupDropdown';
 import {Button} from '@wandb/weave/components/Button';
-import {IconDelete} from '@wandb/weave/components/Icon';
+import {IconDelete, IconPencilEdit} from '@wandb/weave/components/Icon';
 import React, {FC, useState} from 'react';
 import styled from 'styled-components';
 
 import {useClosePeek} from '../../context';
 import {useWFHooks} from '../wfReactInterface/context';
-import {CallSchema} from '../wfReactInterface/wfDataModelHooksInterface';
 
 export const OverflowMenu: FC<{
-  selectedCalls: CallSchema[];
-}> = ({selectedCalls}) => {
+  entity: string;
+  project: string;
+  callIds: string[];
+  callNames: string[];
+  setIsRenaming: (isEditing: boolean) => void;
+}> = ({entity, project, callIds, callNames, setIsRenaming}) => {
   const [confirmDelete, setConfirmDelete] = useState(false);
 
   const menuOptions = [
+    [
```

Comment:
Having this rename button seems to add a lot of complexity and state passing in the code. Most tools just support "click on title to rename" behavior. If we remove this rename option it keeps a simpler UI and simpler code.